omod/src/main/java/org/openmrs/module/patientflags/web/validators/PriorityValidator.java

```diff
@@ -13,33 +13,47 @@
  */
 package org.openmrs.module.patientflags.web.validators;
 
+import org.openmrs.api.context.Context;
 import org.openmrs.module.patientflags.Priority;
+import org.openmrs.module.patientflags.api.FlagService;
 import org.springframework.validation.Errors;
 import org.springframework.validation.ValidationUtils;
 
 /**
  * Validator for the Priority class
  */
 public class PriorityValidator {
-	
+

```

Comment:
thanks @druchniewicz  we can remove the unwanted spaces.,have tested the changes on a running instance ?

Reply:
Hi @HerbertYiga , yes, changes have been tested on a running instance.
regarding unwanted spaces what exactly do you mean? should I re-indent entire file to remove all tabulations and change it to e.g. 4 spaces?